packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -283,20 +290,94 @@ class VotesRepo extends AbstractRepo<"Votes"> {
         }
       }
     }
-    
-    // On EAF, some reacts are anonymous (currently agree and disagree). For those, remove the userId.
-    if (isEAForum) {
-      reactionChanges = reactionChanges.map(change => {
-        if (eaAnonymousEmojiPalette.some(emoji => emoji.name === change.reactionType)) {
-          return {reactionType: change.reactionType}
-        }
-        return change
-      })
-    }
-    
+
     return reactionChanges;
   }
 
+  getEAKarmaChanges({
```

Comment:
It looks like this doesn't respect the `showNegative` arg - I see negative karma notifications when I have it unchecked:
<img width="514" alt="Screen Shot 2024-01-23 at 2 09 47 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/5f147569-5a58-416d-b113-6df0e77129ac">